src/components/ItemsList.js

```diff
@@ -14,6 +15,14 @@ const wasItemPurchasedWithinLastOneDay = (lastPurchasedOn) => {
   return Date.now() - lastPurchasedOn <= oneDayInMilliseconds;
 };
 
+const getDaysBetweenCurrentAndPreviousPurchase = (
+  previousPurchaseDate,
+  currentPurchaseDate = Date.now(),
```

Comment:
Does currentPurchaseDate need to be passed as a parameter? Maybe you can calculate it inside the `getDaysBetweenCurrentAndPreviousPurchase` function :thinking: ?